@navikt/core/react/jest-setup.ts

```diff
@@ -4,3 +4,4 @@ global.ResizeObserver = jest.fn().mockImplementation(() => ({
   unobserve: jest.fn(),
   disconnect: jest.fn(),
 }));
+jest.mock("dialog-polyfill-universal", () => null);
```

Comment:
Må alle som bruker Modal også oppdatere testene sine og mocke polyfill i sine løsninger?

Reply:
Oi, det må de vel mest sannsynlig, ja 😬 Skal prøve å finne en annen løsning. Tips?

Reply:
Hva skjer hvis man ikke mocker polyfill? Vil eks automatiserte tester gjennom browserstack brekke for gamle nettlesere?